tests/IntegrationTests/Git/GitClientTests.cs

```diff
@@ -44,5 +66,57 @@ public void ShouldGetGitLfsVersion()
             var expected = TheVersion.Parse("2.6.1");
             result.Should().Be(expected);
         }
+
+        [Test]
+        public void ShouldCleanFile()
+        {
+            if (!DefaultEnvironment.OnWindows)
```

Comment:
we need tests for Mac as well ( or the same test for both platforms)

Reply:
I'll look into this!